### PR TITLE
Staging v1.11.1 tasks

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Review release tasks for v1.11.1, this typically are reviewed in the merge against main, but since this is just a patch then this is just to have an extra set of eyes on this.

##### ISSUE TYPE

- Release Pull Request
- 
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by fernandofloresg** [comment]
This shows ansible sanity failing but appears to be an issue with most recent PRs as you can see in the details is not related to the code. Executed it locally and does not fail, also, uploading this to the test servers works fine too.


```bash
(venv-2.17) ./ac --ac-sanity
-----------------------------------------------------------------------
Running ansible-test with managed python virtual environment: /Users/fernandoflores/Documents/IBM/Repos/ibm_zos_core/venv/venv-2.17.
-----------------------------------------------------------------------
Running sanity test "action-plugin-docs"
Running sanity test "ansible-doc"
Running sanity test "changelog"
Running sanity test "compile" on Python 3.12
Running sanity test "empty-init"
Running sanity test "ignores"
Running sanity test "import" on Python 3.12
Running sanity test "line-endings"
Running sanity test "no-assert"
Running sanity test "no-get-exception"
Running sanity test "no-illegal-filenames"
Running sanity test "no-smart-quotes"
Running sanity test "pep8"
Running sanity test "pslint"
Running sanity test "pylint"
Running sanity test "replace-urlopen"
Running sanity test "runtime-metadata"
Running sanity test "shebang"
Running sanity test "shellcheck"
Running sanity test "symlinks"
Running sanity test "use-argspec-type-path"
Running sanity test "use-compat-six"
Running sanity test "validate-modules"
Running sanity test "yamllint"

```

File: README.md [hunk 1]
```diff
@@ -174,14 +174,15 @@ For Galaxy and GitHub users, to see the supported ansible-core versions, review
...
-| 1.7.x    | Released       | [Release notes](https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html#version-1-7-0)    | [Changelogs](https://github.com/ansible-collections/ibm_zos_core/blob/v1.7.0/CHANGELOG.rst)  |
-| 1.6.x    | Released       | [Release notes](https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html#version-1-6-0)    | [Changelogs](https://github.com/ansible-collections/ibm_zos_core/blob/v1.6.0/CHANGELOG.rst)  |
-| 1.5.x    | Released       | [Release notes](https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html#version-1-5-0)    | [Changelogs](https://github.com/ansible-collections/ibm_zos_core/blob/v1.5.0/CHANGELOG.rst)  |
+| 1.13.x   | In development | unreleased    | unreleased |
+| 1.12.x   | Current   | [Release notes](https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html#version-1-11-0)   | [Changelogs](https://github.com/ansible-collections/ibm_zos_core/blob/v1.12.0-beta.1/CHANGELOG.rst#v1-12-0-beta-1) |
```
Comment (rexemin) [inline README.md:178]: You are linking to v1.11.0's release notes instead of v1.12.0's

**Review by rexemin — CHANGES_REQUESTED** [review]
Looks good, just noticed one wrong link

File: README.md (hunk 1 above)
Comment (fernandofloresg) [reply README.md:178]: great that you noticed !

**Review by rexemin — APPROVED** [review]

**Review by AndreMarcel99 — APPROVED** [review]

**Review by richp405 — COMMENTED** [review]
lgtm

**Pull request merged**